packages/rancher-istio/charts/Chart.yaml

```diff
@@ -16,3 +16,5 @@ annotations:
   catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.29.000-rc00
   catalog.cattle.io/display-name: "Istio"
   catalog.cattle.io/os: linux
+  catalog.cattle.io/requests-cpu: "4500m"
+  catalog.cattle.io/requests-memory: "5620Mi"
```

Comment:
Curious how you came to these numbers ?

Reply:
I went into each chart in the manifest files and looked at their values.yaml default resource settings, for example this is from the ingress gateway chart
```
    resources:
      requests:
        cpu: 100m
        memory: 128Mi
      limits:
        cpu: 2000m
        memory: 1024Mi
```
And I added them together to get a minimum default setting - I did this when we initially switch to V2 https://rancher.com/docs/rancher/v2.x/en/istio/v2.5/resources/ - but now that I am looking at this side by side - something is wack... somehow...columns got switched? I will quickly do this again and get the PR updated as well as updates to the Docs

Reply:
TLDR; I mixed up values for limits vs requests during calculations so they were incorrect

When making original calculations, some of the resources had switched the order in which they listed the resources requests and limits so I also did this in many places. 

I have redone the calculations, this no longer includes removed resources like mixer etc. These values were pulled from the default values for the resource https://github.com/rancher/docs/issues/2999